wordpress-mobile/release-toolkit
# Use Sodium for secret encryption and decryption

Fixes an openSSL warning:

```
openssl aes-256-cbc -md sha256 -d -in .circleci/.firebase.secrets.json.enc -out .circleci/.firebase.secrets.json -k “${FIREBASE_SECRETS_ENCRYPTION_KEY}”
*** WARNING : deprecated key derivation used.
Using -iter or -pbkdf2 would be better.
```

Also gets us out of manual crypto.

This is a breaking change and should be carefully merged to avoid breaking projects.

**To Test**
1. Check out the code and make sure it looks ok (including test changes)
2. Ensure that CI passes
